src/main/resources/config/liquibase/changelog/20190828140700_change_task_data_varchar.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <property name="textType" value="varchar" dbms="postgresql, h2"/>
+    <property name="textType" value="varchar(1024)" dbms="oracle"/>
```

Comment:
i think it is not enough 1024 bytes for `data` column. Maybe we should choose other unbound type?

Reply:
changed to 400